Solutions/CohesitySecurity/Tests/az.test.py

```diff
@@ -46,6 +52,49 @@ def setUp(self):
         if error:
             print("error --> %s" % error.decode())
 
+    def test_cohesity_restore_from_last_snapshot(self):
+        """
+        Test the Cohesity_Restore_From_Last_Snapshot playbook.
+        """
+        print("Starting test_cohesity_restore_from_last_snapshot...")
+        playbook_name = "Cohesity_Restore_From_Last_Snapshot"
+        subscription_id = get_subscription_id()
+        result = search_alert_id_in_incident(
+            self.alert_id, self.resource_group, self.workspace_name)
+        jsObj = result[0]
+        incident_id = jsObj["id"].split("/")[-1]
+
+        alert_details = get_alert_details(self.alert_id, self.api_key)
+        alert = Alert(json.dumps(alert_details))
+        protection_group_id = alert.get_protection_group_id()
+        cluster_id = alert.get_cluster_id()
+        current_time_usecs = int(time.time() * 1000000)
+        start_time_usecs = current_time_usecs - 180000000
```

Comment:
Please add a comment that you're fetching only the recoveries that start in the last 30 min.

In fact, based on the validation you're doing which expects that there are no such recoveries with the same protection group ID, please use 10 min.

Reply:
done, and current time is 3 min, is this ok?

Reply:
3 min is fine. The 0s got me confused, hence the need for a comments. Consider doing something like

Consider creating a constant like microSecsInSec = 1000000 and rewrite this as 

start_time_usecs = start_time_usecs - (180 * microSecsInSec)

Reply:
done